.env.example

```diff
@@ -35,6 +35,10 @@ GBP_RATE=
 RAW_POST_URL=
 JSON_POST_URL=
 
+# Id eSkarbonki WOŚP
```

Comment:
Nitpick - rzeczy takie jak ID, DB, SQL piszemy wielkimi.
Przydałyby się też może 2-3 zdania w readme jak to ID pozyskać.

```suggestion
# ID eSkarbonki WOŚP
```